### ISSUE TITLE
BUG: Dragging with Alt (to duplicate) an object with a connected arrow makes arrow lose connection and position

### ISSUE THREAD

**Issue by n-konig** [zsviczian/obsidian-excalidraw-plugin#1455]
**Your environment**
SYSTEM INFO:
	Obsidian version: v1.5.0
	Installer version: v1.4.13
	Operating system: Windows 10 Education 10.0.19045
	Login status: logged in
	Catalyst license: insider
	Insider build toggle: on
	Live preview: on
	Base theme: light
	Community theme: none
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 2
	Plugins enabled: 2
		1: Webpage HTML Export v1.7.3
		2: Excalidraw v2.0.3

RECOMMENDATIONS:
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.

**Describe the bug**
When dragging an object with the Alt key pressed, this duplicates the object - but it also displaces the original object based on when the Alt key was pressed, and makes any arrows connected to the original object lose their connection.

**To Reproduce**
1. Draw two rectangles and connect them with an arrow
2. Click and drag one of the rectangles
3. Press the Alt key to duplicate the rectangle
4. Keep dragging while holding the Alt key pressed, and then release somewhere

**Expected behavior**
As soon as the Alt key is pressed during a click-and-drag, the object(s) being dragged should return to the original position and the object(s) being dragged should be duplicated.

**Screenshots**

![excdragduplicatebug](https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/104557082/055b5bd1-3ecd-4755-9bb5-0dc446934b5c)

**Additional context**
I LOVE EXCALIDRAW

**Comment by zsviczian** [comment]
Yes, this is frustrating. Can I ask you to open an issue with the main excalidraw package. This should be fixed there, not in the Obsidian plugin.

https://github.com/excalidraw/excalidraw/issues
